File: app/src/app/core/models/grade.ts
/*
 * Este fichero forma parte de UBUMonitorWeb.
 *
 * @author Marcelo Verteramo Pérsico
 */

import { GradeItem } from './grade-item';

export type Grade = {
  courseId: number;
  userId: number;
  grades: GradeItem[];
};

File: api/src/main/kotlin/es/ubu/lsi/ubumonitorweb/domain/Grade.kt
/*
 * Este fichero forma parte de UBUMonitorWeb.
 *
 * @author Marcelo Verteramo Pérsico
 */

package es.ubu.lsi.ubumonitorweb.domain

data class Grade(
  val courseId: Long,
  val userId: Long,
  val grades: List<GradeItem>,
)
